File: Python/BarangElektronik.py
class BarangElektronik:
    def __init__(self, id=-1, nama="", manufaktur="", harga=0):
        self.__id = id
        self.__nama = nama
        self.__manufaktur = manufaktur
        self.__harga = harga

    # Getter & Setter
    def get_id(self):
        return self.__id

    def set_id(self, id):
        self.__id = id

    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    def get_manufaktur(self):
        return self.__manufaktur

    def set_manufaktur(self, manufaktur):
        self.__manufaktur = manufaktur

    def get_harga(self):
        return self.__harga

    def set_harga(self, harga):
        self.__harga = harga

    def tampilkan_info(self):
        print(f"ID: {self.__id}")
        print(f"Nama: {self.__nama}")
        print(f"Manufaktur: {self.__manufaktur}")
        print(f"Harga: Rp{self.__harga}")

File: Java/BarangElektronik.java
public class BarangElektronik {
    private int id;
    private String nama;
    private String manufaktur;
    private int harga;

    // Constructor default
    public BarangElektronik() {
        this.id = -1;
        this.nama = "";
        this.manufaktur = "";
        this.harga = 0;
    }

    // Constructor dengan parameter
    public BarangElektronik(int id, String nama, String manufaktur, int harga) {
        this.id = id;
        this.nama = nama;
        this.manufaktur = manufaktur;
        this.harga = harga;
    }

    // Getter dan Setter
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNama() {
        return nama;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public String getManufaktur() {
        return manufaktur;
    }

    public void setManufaktur(String manufaktur) {
        this.manufaktur = manufaktur;
    }

    public int getHarga() {
        return harga;
    }

    public void setHarga(int harga) {
        this.harga = harga;
    }

    public void tampilkanInfo() {
        System.out.println("ID: " + id);
        System.out.println("Nama: " + nama);
        System.out.println("Manufaktur: " + manufaktur);
        System.out.println("Harga: Rp" + harga);
    }
}
